bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
btw, what about `external equal : t -> t -> bool = "caml_string_equal"` used in bytes.ml ?